packages/matter.js/tsconfig.base.json

```diff
@@ -1,9 +1,7 @@
 /* Base configuration */
 {
-  "extends": "../../tsconfig.json",
```

Comment:
by roming this you disable all the ts checks coming from the base one for building. Is this really intended?

Reply:
Yes:
1 _ tsconfig.base.json is the common ts config for the package
2 _ tsconfig.dist[*].json are the ts configs for building the distribution
3 _ tsconfig.json is the ts config for the local development and testing

ts checks are only useful for local development since testing will already enforce them, so no need to declare them again for building the distribution (it just slows down the build for no benefits)